tests/test_berry.py

```diff
@@ -67,24 +72,40 @@ def compare_quant(quant):
         return quant
 
 
-
-def test_Fe(check_integrate, system_Fe_W90, compare_energyresult,quantities_Fe,Efermi_Fe):
+def test_Fe(check_integrate,system_Fe_W90, compare_energyresult,quantities_Fe,Efermi_Fe):
     """Test anomalous Hall conductivity , ohmic conductivity, dos, cumdos"""
     check_integrate(system_Fe_W90 , quantities_Fe , fout_name="berry_Fe_W90" , suffix="" , Efermi=Efermi_Fe , comparer=compare_energyresult )
 
 
-def test_Fe_wcc(check_integrate, system_Fe_W90_wcc, compare_energyresult,quantities_Fe,Efermi_Fe):
+def test_Fe_wcc(check_integrate,system_Fe_W90_wcc, compare_energyresult,quantities_Fe,Efermi_Fe):
```

Comment:
generally, lines that are in fact not changes, should not show as changed. Unless you are doing a 'cosmetic' commit.